students/Nick_Lenssen/lesson03/slicing.py

```diff
@@ -0,0 +1,32 @@
+def first_last(seq):
+    return (seq[-1:]+seq[1:-1]+seq[:1])
+
+def every_other(seq):
+    return (seq[0::2])
+
+def remove_high_low(seq):
+    return (seq[4:-4])
```

Comment:
"with the first 4 and the last 4 items removed, and then every other item in the remaining sequence."

should be `seq[4:-4:2]`